src/pages/patterns/notification-pattern/index.mdx

```diff
@@ -142,19 +143,24 @@ experience for users.
 
 <Title> Deciding what to use </Title>
 
-| Status        | Usage                                                                                                                        | Action                                                                                                         | Color  | Icon                 |
-| ------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- | ------ | -------------------- |
-| Informational | Provide additional information to users that may not be tied to their current action or task                                 | Do not require immediate action and can be dismissed on a timer or persist, depending on the content           | Blue   | `information filled` |
-| Success       | Confirm a task was completed as expected                                                                                     | Typically do not require further action and can be dismissed automatically or persist in a nonintrusive manner | Green  | `checkmark filled`   |
-| Warning       | Inform users that they are taking actions that are not desirable or might have unexpected results                            | Often persist until the user dismisses the notification or continues in their task                             | Yellow | `warning filled`     |
-| Error         | Inform users of an error or critical failure and optionally block the user from proceeding until the issue has been resolved | Always persist until the user dismisses them or resolves error                                                 | Red    | `error filled`       |
+Notifications typically serve as feedback after an action or task is completed.
+However, the callout variant is an exception, as it is used to provide guidance
+before users begin a task or take an action and, therefore, does not include
+success or error statuses.
+
+| Status        | Usage                                                                                                                        | Color  | Icon                 | Variants                           |
```

Comment:
I see you added "variants" to the table but didn't include the other "types" of pattern notifications like modal, banner, and notification panel. The pattern page may not need that last column.